transcribed/page1170.txt

```diff
@@ -0,0 +1,35 @@
+001                                                                             PAGE 117
+002
+003     003 004 SUBTTL RANDOM STORAGE
+004
+005             RADIO 10.
```

Comment:
RADIX

Reply:
Nope! I like the radio better! But you're right of course... :)